### ISSUE TITLE
VSCode Debugging broke after upgrading from version 9.2.2 to 9.3.0

### ISSUE THREAD

**Issue by chadalen** [zeit/next.js#11095]
# Bug report
Debugging through VSCode no longer works

## Describe the bug
When running debugger through VSCode console after upgrading from version 9.2.2 to 9.30 the console spams
"Starting inspector on 127.0.0.1:9229 failed: address already in use" but the address is not in use.

```
baydash on master [!] is 📦 v1.0.0 via ⬢ v12.13.0 on ☁️  us-east-2 
❯  cd /Users/crazypanda/Workspaces/baydash ; env NODE_OPTIONS=--inspect-brk /usr/local/bin/node node_modules/.bin/next 
Debugger listening on ws://127.0.0.1:9229/61643430-1b26-4c6b-864e-6413d4daf6b9
For help, see: https://nodejs.org/en/docs/inspector
Debugger attached.
[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
Starting inspector on 127.0.0.1:9229 failed: address already in use
Starting inspector on 127.0.0.1:9229 failed: address already in use
Starting inspector on 127.0.0.1:9229 failed: address already in use
Starting inspector on 127.0.0.1:9229 failed: address already in use
```

## To Reproduce
1. Use Next.js version 9.3.0
2. Create a Next.js app
3. Create a launch.json file (for VSCode)
```javascript
{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "node",
            "request": "launch",
            "name": "Debug App",
            "program": "${workspaceFolder}/node_modules/.bin/next",
            "port": 9229,
            "console": "integratedTerminal",
            "cwd": "${workspaceFolder}",
            "env": {
                "NODE_OPTIONS": "--inspect-brk"
            }
        }
    ]
}
```
In VSCode run the debugger
![image](https://user-images.githubusercontent.com/12568665/76719179-f8366300-6706-11ea-8484-4601d510ea89.png)

## Expected behavior
Should not say "Starting inspector on 127.0.0.1:9229 failed: address already in use". The server should run and you should be able to set breakpoints in VSCode.

![image](https://user-images.githubusercontent.com/12568665/76719363-988c8780-6707-11ea-874f-3992b73bad99.png)

## System information

- OS: MacOS
- Browser (if applies) Microsoft Edge (Chromium)
- Version of Next.js: 9.3.0

## Additional context
This does not happen in version 9.2.2

**Comment by pecoram** [comment]
same issue :cry:

**Comment by pecoram** [comment]
any news about this issue?

**Comment by chadalen** [comment]
@pecoram No unfortunately.

@timneutkens  @Timer

**Comment by chadalen** [comment]
Gonna close this ticket seems like `--inspect` node.js flag gets called underneath.

VSCode debugging works with this config for me.
```javascript
{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "node",
            "request": "launch",
            "name": "Debug App",
            "program": "${workspaceFolder}/node_modules/.bin/next",
            "console": "integratedTerminal",
            "cwd": "${workspaceFolder}"
        }
    ]
}
```

**Comment by haneatic** [comment]
@chadalen and what do I do if I use TypeScript? Placing a breakpoint shows a warning that SourceMaps aren't found.

**Comment by JoseFMP** [comment]
Same issue here. Wondering, any plans to fix this?

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
